### PR TITLE
Introduce Photon Ant

### PR DESCRIPTION
**Pull request by andymikulski**
- Adds `ant-design` subdirectory
- Adds `photon-ant` NPM package.json (currently NOT published)
- Adds basic Photon styles for Ant components

Current usage is simply adding `photon-ant` as a dependency and then dropping `@import '~photon-ant/index.less';` into a LESS file. There is still some more work to be done, but I wanted to get this looked at before the PR got much bigger!

### CODE REVIEW

**Comment by andymikulski** [comment]
Note there's also a [living style guide](https://photon-design.neocities.org/components/button/) component to the photon-ant business. The site is good for visually checking progress, however the code to keep the guide standing on its own is a little hairy at this moment. It will be introduced in a later PR.

File: ant-design/src/theme-overrides.less [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+@link-color             : @primary-color;
+@link-hover-color       : @primary-5;
+@link-active-color      : @primary-7;
+@link-hover-decoration  : none;
+
+// Animation
+@photon-easing          : cubic-bezier(.07,.95,0,1);
+
+// Border color
+@border-color-base      : #d9d9d9;        // base border outline a component
```
Comment (bwinton) [inline ant-design/src/theme-overrides.less]: Is this a non-Design-System colour? And if so, can we make it a Design-System colour?

Comment (andymikulski) [reply ant-design/src/theme-overrides.less]: The issue I had was that Ant didn't like using an `rgba` value for this variable, as it is used later in a function, and I guess the function doesn't like `rgba`s.

I've corrected this to now use the appropriate color/opacity, and appropriately adjusted the _other_ variable that relied on this not having opacity!

**Comment by andymikulski** [comment]
I'm going to land this so we can publish v0.1.0.

**Pull request merged**